src/iht.jl

```diff
@@ -42,6 +42,7 @@ function L0_reg(
     debias    :: Bool = false,
     show_info :: Bool = true,          # print things when model didn't converge
     init      :: Bool = false,         # not efficient. whether to initialize β to sensible values
+    est_r     :: Int = 0,              # If 1, estimate r using MM algorithm. If 2, estimate r using Newton's Method
```

Comment:
Using integer to represent which algorithm to run is not intuitive. A better alternative is to use Symbol in Julia. For instance, :MM and :Newton.